disputer/index.js

```diff
@@ -132,6 +170,12 @@ async function Poll(callback) {
     // Default to 1 minute delay. If set to 0 in env variables then the script will exit after full execution.
     const pollingDelay = process.env.POLLING_DELAY ? Number(process.env.POLLING_DELAY) : 60;
 
+    // Default to 3 re-tries on error within the execution loop.
+    const errorRetries = process.env.ERROR_RETRIES ? Number(process.env.ERROR_RETRIES) : 5;
+
+    // Default to 10 seconds in between error re-tries.
+    const errorRetriesTimeout = process.env.ERROR_RETRIES__TIMEOUT ? Number(process.env.ERROR_RETRIES__TIMEOUT) : 10000;
```

Comment:
env slightly confusing to be set in MS while everything else is in seconds. maybe have use specify in seconds and convert to MS to be used internally

Reply:
ye, it's a good point. I like using second throughout. will add your suggestion.